geonode_dominode/geonode_dominode/tasks.py

```diff
@@ -0,0 +1,69 @@
+from io import StringIO
+
+from celery.utils.log import get_task_logger
+from django.conf import settings
+from django.contrib.auth import get_user_model
+from django.core import management
+from django.core.mail import send_mail
+from django.template.loader import render_to_string
+from django.utils.translation import ugettext_lazy as _
+from geonode.geoserver.helpers import gs_slurp
+
+from geonode_dominode.celeryapp import app
+
+logger = get_task_logger(__name__)
+
+
+@app.task(queue='geonode_dominode')
+def task_cli_sync_geoserver(workspace_name, username):
+    # construct management command arguments
+    out = StringIO()
+    try:
+        result = gs_slurp(
+            ignore_errors=False,
+            verbosity=1,
+            owner=username,
```

Comment:
Beware - this parameter is supposed to be a proper django user, not a string.

Reply:
done